### PR TITLE
Deprecate Pixlr integration

### LINKED ISSUE #1092: Remove inline scripts
For many reasons, we should remove all scripts from the templates. We could replace them with data listerns like in the page bundle: https://github.com/sonata-project/SonataPageBundle/pull/562.
## General

Inline scripts couldn't be cached and would pollute the html output.
## Security

There is also a [security bundle](https://github.com/nelmio/NelmioSecurityBundle#content-security-policy) which introduces some interesting information about security. You could completly disable inline script via a http header to reduce possible XSS attacks. 

For more information: http://www.w3.org/TR/CSP/
## Todo

The following files contains inline scripts and should be removed in the next major release, because this is a BC break.
- [ ] https://github.com/sonata-project/SonataMediaBundle/blob/master/Resources/views/MediaAdmin/edit.html.twig
- [ ] https://github.com/sonata-project/SonataMediaBundle/blob/master/Resources/views/Extra/pixlr_exit.html.twig

### LINKED ISSUE #2107: Deprecate Pixlr integration?
## Deprecation Request

Should we deprecate Pixlr integration? I am not even sure if it works or is broken (it is a really old integration and the Pixlr webpage changed a lot).

Does it make sense to have this kind of feature here or should we focus on image upload / download?

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this is BC.

Closes #2107 .
Closes #1092 .

## Changelog

```markdown
### Deprecated
- Deprecated Pixlr integration.
```

### CODE REVIEW

**Review by core23 — DISMISSED** [review]

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -446,6 +446,10 @@ private function addExtraSection(ArrayNodeDefinition $node): void
         $node
             ->children()
                 ->arrayNode('pixlr')
+                    ->setDeprecated(...$this->getBackwardCompatibleArgumentsForSetDeprecated(
+                        'The node "%node%" is deprecated and will be removed in version 4.0.',
+                        '3.28'
```
Comment (franmomu) [inline src/DependencyInjection/Configuration.php:451]:
```suggestion
                        '3.x'
```

Comment (VincentLanglet) [reply src/DependencyInjection/Configuration.php:451]: And should we add a NEXT_MAJOR comment ?

Comment (jordisala1991) [reply src/DependencyInjection/Configuration.php]: Fixed.

**Review by core23 — CHANGES_REQUESTED** [review]
You can also add a deprecation warning for the template

**Review by core23 — APPROVED** [review]

**Review by VincentLanglet — APPROVED** [review]

**Review by franmomu — APPROVED** [review]

**Comment by franmomu** [comment]
Thanks @jordisala1991 !

**Pull request merged**